synapse/models/geopol.py

```diff
@@ -14,12 +14,12 @@ def getModelDefs(self):
                     ),
 
                     ('pol:iso2',
-                        ('str', {'lower': True, 'regex': '^[a-z0-9]{2}$', 'nullval': '??'}),
```

Comment:
Isn't this a backward incompatible change? Also, why in this PR?

Reply:
nullval is not used anywhere actually. its an artifact of past implementation ideas that has real no effect.

Reply:
This was also one of the existing model elements I was originally testing things with.